doc/examples.md

```diff
@@ -0,0 +1,50 @@
+The following code adds two vectors:
+```cpp
+#include <cassert>
+#include <iostream>
+
+#include <CL/sycl.hpp>
```

Comment:
hm... `CL/sycl.hpp` is still the only valid way according to the SYCL spec since SYCL 2020 is not out yet (2020 provisional is after all more of a preview for developers)

Reply:
Ah okay that makes sense.